product-scenarios/1-integrating-systems-that-communicate-in-heterogeneous-message-formats/01-SynapseConfigProject/01-synapseConfig/artifact.xml

```diff
@@ -71,6 +71,11 @@
     <artifact name="1_6_2_4_Proxy_Add_Sibling_toMessageBodyStoredInProperty" groupId="org.wso2.ei.product-scenarios.scenario_01.proxy-service" version="1.0.0" type="synapse/proxy-service" serverRole="EnterpriseServiceBus">
         <file>src/main/synapse-config/proxy-services/1_6_2_4_Proxy_Add_Sibling_toMessageBodyStoredInProperty.xml</file>
     </artifact>
+    <artifact name="1_6_10_1_Proxy_AlterPayloadWithInlineGroovyScript" groupId="org.wso2.ei.product-scenarios.scenario_01.proxy-service" version="1.0.0" type="synapse/proxy-service" serverRole="EnterpriseServiceBus">
+        <file>src/main/synapse-config/proxy-services/1_6_10_1_Proxy_AlterPayloadWithInlineGroovyScript.xml</file>
+    </artifact>
+    <artifact name="1_6_10_2_Proxy_AlterPayloadWithInlineJavaScript" groupId="org.wso2.ei.product-scenarios.scenario_01.proxy-service" version="1.0.0" type="synapse/proxy-service" serverRole="EnterpriseServiceBus">
+        <file>src/main/synapse-config/proxy-services/1_6_10_2_Proxy_AlterPayloadWithInlineJavaScript.xml</file>
```

Comment:
`"</artifact>"` is missing here. Shall we fix this?

Reply:
Done